pkgs/applications/misc/blender/default.nix

```diff
@@ -7,6 +7,7 @@
 , jackaudioSupport ? false, libjack2
 , cudaSupport ? config.cudaSupport or false, cudatoolkit
 , colladaSupport ? true, opencollada
+, denoiseSupport ? true, openimagedenoise
```

Comment:
Instead of adding a new option, why not just compile with `openimagedenoise` unconditionally? The impact on the closure size is not that big:
```
before:
/nix/store/ad9c14dbyb8njkywaxzsyaj3n2yfpsgk-blender-2.82a        547.0M
after:
/nix/store/78cmqhwpsgvw7yx1kaqksx5y1rn1795h-blender-2.82a        592.7M
```

One can always use an override `blender = blender.override { openimagedenoise = null; }` to disable it.

Reply:
OK. Just wanted to be consistent with other options.